modules/apps/callbacks/testing/simapp/ante_handler.go

```diff
@@ -1,37 +1,43 @@
 package simapp
 
 import (
-	errorsmod "cosmossdk.io/errors"
+	"errors"
+
+	circuitante "cosmossdk.io/x/circuit/ante"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/ante"
 
 	ibcante "github.com/cosmos/ibc-go/v7/modules/core/ante"
-	ibcerrors "github.com/cosmos/ibc-go/v7/modules/core/errors"
 	"github.com/cosmos/ibc-go/v7/modules/core/keeper"
 )
 
-// HandlerOptions extend the SDK's AnteHandler options by requiring the IBC keeper.
+// HandlerOptions are the options required for constructing a default SDK AnteHandler.
 type HandlerOptions struct {
 	ante.HandlerOptions
-
-	IBCKeeper *keeper.Keeper
+	CircuitKeeper circuitante.CircuitBreaker
```

Comment:
Sure I can remove. I felt the same initially, as it isn't strictly required, but we should maybe consider adding circuit breaker e2e tests

Reply:
As per offline discussion, we decided to open an issue and decide later since it is a bit opinionated to make this change without removing other unnecessary modules

Reply:
https://github.com/cosmos/ibc-go/issues/4552